extensions/virtuallistview/src/VirtualListCell.tsx

```diff
@@ -323,30 +340,48 @@ export class VirtualListCell<ItemInfo extends VirtualListCellInfo> extends RX.Co
 
         return (
             <RX.Animated.View
-                style={ [_styles.cellView, overflow, this._animatedStylePosition] }
+                style={ [_styles.cellView, overflow, this._animatedStylePosition, this._animatedStyleWidth] }
                 ref={ _virtualCellRef }
                 tabIndex={ this.props.tabIndex }
                 onLayout={ this.props.onLayout ? this._onLayout : undefined }
                 onFocus={ this.props.onCellFocus ? this._onFocus : undefined }
                 onBlur={ this.props.onCellFocus ? this._onBlur : undefined }
+                onPress={this.props.onItemSelected ? this._onPress : undefined}
+                onKeyPress={ this.props.onKeyPress || this.props.onItemSelected ? this._onKeyPress : undefined}
             >
                 <VirtualListCell.StaticRenderer
                     shouldUpdate={ this.props.shouldUpdate }
                     isFocused={ this.props.isFocused }
-                    style={ [overflow, this._animatedStyleWidth] }
+                    isSelected={ this.props.isSelected }
                     item={ this.props.item }
                     renderItem={ this.props.renderItem }
                 />
             </RX.Animated.View>
         );
     }
 
+    private _onKeyPress = (e: RX.Types.KeyboardEvent) => {
+        const isSelectItemKeyPress = e.keyCode === _keyCodeEnter || e.keyCode === _keyCodeSpace;
+        if (isSelectItemKeyPress && this.props.onItemSelected && this.props.item) {
+            this.props.onItemSelected(this.props.item);
+        }
+        if (this.props.onKeyPress) {
+            this.props.onKeyPress(e);
+        }
+    }
+
     private _onFocus = (e: RX.Types.FocusEvent) => {
         if (this.props.onCellFocus) {
             this.props.onCellFocus(this.props.itemKey);
         }
     }
 
+    private _onPress = (e: RX.Types.SyntheticEvent) => {
+        if (this.props.onItemSelected && this.props.item) {
+            this.props.onItemSelected(this.props.item);
```

Comment:
Same comment as above. You're not marking the event as handled, so it will continue to bubble up the view hierarchy. Is that what you want?